observer/worker_track.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+""" Class to facilitate tracking loop.
+"""
+
+from . import commsocket
```

Comment:
Our current master branch has a ``setup.py`` in order to maintain our code and easily create a package out of it. Please use absolute URLs.